pkgs/games/mindustry/default.nix

```diff
@@ -32,20 +32,20 @@
 
 let
   pname = "mindustry";
-  version = "140.4";
+  version = "141.2";
   buildVersion = makeBuildVersion version;
 
   Mindustry = fetchFromGitHub {
     owner = "Anuken";
     repo = "Mindustry";
     rev = "v${version}";
-    sha256 = "0lfDISvC5dOd76BuPPniPf+8hXWQ/wtreNHN75k2jVA=";
+    sha256 = "sha256-7olnyjkcT8OwokipDnLFW3rMOPljF6HvsU249SDvA3U=";
```

Comment:
I think this should be changed to `hash` since you're using SRI, but it'll be accepted by the fetcher anyway. I can change all of them in another pr